astropy/units/tests/test_quantity_erfa_ufuncs.py

```diff
@@ -542,7 +549,11 @@ def setup_class(self):
     def test_unit_errors(self):
         """Test unit errors when dimensionless parameters are used"""
 
-        msg = "'NoneType'.*no attribute 'get_converter'.*\n.*treated as dimensionless"
+        msg = "'NoneType' object has no attribute 'get_converter'"
+        if not (PYTEST_LT_8_0 and sys.version_info >= (3, 11)):
+            # For python>=3.11, we use Exception.add_note, which
+            # pytest < 8 does not know how to deal with.
+            msg += ".*\n.*treated as dimensionless"
```

Comment:
My regex-fu isn't that strong but I thought usually having a `\` in the match needs raw string?

Reply:
Here, I do want the `\n` be interpreted as new-line. Happy it worked!